corehq/apps/hqcase/management/commands/reprocess_form_case_es_deletes.py

```diff
@@ -0,0 +1,90 @@
+from datetime import datetime, timedelta
+from django.core.management import BaseCommand, CommandError
+
+from corehq.apps.hqadmin.models import HistoricalPillowCheckpoint
+from pillowtop import get_all_pillow_instances
+from pillowtop.models import str_to_kafka_seq, KafkaCheckpoint
+from pillowtop.processors.elastic import ElasticProcessor
+from pillowtop.utils import get_pillow_by_name
+
+
+def get_active_pillows(pillows):
+    # return active pillows based on the heuristic 
+    #   that have their checkpoints updated in last 30 days
+    active_pillows = []
```

Comment:
It will be nice to get a second eye on this logic that decides which pillows we need to reprocess the deletes on. 

The goal of this command is to make sure the deletes in Form/Case ES indices get processed because these deletes are not handled in the reindex command which only copies data across but doesn't delete the data. The logic I followed is this
1. Get a list of active pillows that update form/case ES 
2. Different envs may use different pillows, so an active pillow is inferred based on its checkpoints last modification time.
3. Only deletion part of the pillow.process_change is triggered based on whether a pillow processor is ElasticProcessor or not

Reply:
I don't have enough context to evaluate this. Maybe @proteusvacuum could weigh in?

Reply:
@calellowitz  or @snopoke ?

Reply:
Logic seems OK to me. You could further filter by topic (unless that's happening somewhere else).

Reply:
All the topics should get processed per this line https://github.com/dimagi/commcare-hq/pull/28450/files#diff-5906407fba48bdc631775d7847e02a3aR81 below

Reply:
Also the model doesn't have a topic column to filter individually on. https://staging.commcarehq.org/admin/hqadmin/historicalpillowcheckpoint/